client/me/help/help-contact/index.jsx

```diff
@@ -228,10 +222,15 @@ module.exports = React.createClass( {
 		}
 
 		if ( olark.isOperatorAvailable ) {
-			return this.getChatForm();
+			contactFormProps = this.getChatFormProperties();
+		} else {
+			contactFormProps = this.getKayakoFormProperties();
 		}
 
-		return this.getKayakoTicketForm();
+		// Hide the olark widget in the bottom right corner.
+		olarkActions.hideBox();
```

Comment:
This is just a DOM manipulation for a global element, right? Just checking that we aren't attempting to change the component's state mid-render.